src/itemMembership/factory.ts

```diff
@@ -12,7 +12,7 @@ export const ItemMembershipFactory = (
   id: faker.string.uuid(),
   createdAt: faker.date.anytime().toISOString(),
   updatedAt: faker.date.anytime().toISOString(),
-  member: MemberFactory(im.member),
+  account: im.account || MemberFactory(),
```

Comment:
Shouldn't this be `AccountFactory` ? Because `MemberFactory` return a `Member` right ?

Reply:
Unfortunately, ItemMembership need either a `type` Member or Guest. AccountFactory will not give the `type` property